### ISSUE TITLE
Additional support for file import

### ISSUE THREAD

**Issue by Ron024** [colour-science/colour#259]
Add support for JCAMP-DX and CxF XML imports.

**Comment by KelSolaar** [comment]
Hi @Ron024, any more details / specs on those file formats?

**Comment by Ron024** [comment]
Yes, I have been working on adding JCAMP support and was going take a stab at the CxF format.

**Comment by KelSolaar** [comment]
References
---------------
- http://www.jcamp-dx.org/
- http://www.xrite.com/documents/literature/CxF/CxF_Standard.pdf

**Comment by Ampersandme** [comment]
Hi, I was curious to where this had got to? As I have a interest, currently trying import and parse .mif , .cxf , .qtx , .jb5 file formats to use with the Colour package. My current version of xrite iQC supports exports in these formats unless there is a trick to convert to .txt that I do not know. Apologies if it's a bit noob, still learning. 
Cheers.

**Comment by KelSolaar** [comment]
Hi @Ampersandme,

I haven't work on this at all not sure if @Ron024 did any progress on that either.

Cheers,

Thomas

**Comment by Ron024** [comment]
I have some jcamp implementation using another library. Since there is intrest in this feature I will work on adding CxF support.

**Comment by Ron024** [comment]
@Ampersandme if you copy the measurement to the clipboard by right clicking on left hand side where they are all displayed and picking copy sample to clipboard. You can then paste it into excel and format it to use the CSV import pretty easily.

**Comment by Ampersandme** [comment]
@Ron024 Hi! Thank you, yeah I have interest in this feature but that interest is not specific to the format .exp more so in extracting more of the raw information from the .ejob itself when read as a .txt file or .csv . The formats I listed seem to pull relevant information out of the ejob for an intended purpose i.e. imaging software. Currently for my purposes the average that went into the initial measurement is of interest, if specifically that information can be extracted or was retained. 

That CSV import method sounds great! I will most certainly use it in the future, as I acquaint myself with Colour further. I will happily also offer a solution too if I find one.
